server/games/game.py

```diff
@@ -390,6 +400,11 @@ async def remove_game_connection(self, game_connection):
         if self.state is GameState.LOBBY and player.id in self._player_options:
             del self._player_options[player.id]
 
+        if not self._launch_future.done() and (
+                self.state is GameState.INITIALIZING or self.state is GameState.LOBBY
```

Comment:
Extra indent

Reply:
Could also do `self.state in (GameState.INITIALIZING, GameState.LOBBY)`. Might make it cleaner?

Reply:
I also wonder if it might be better to move all of the `wait_*` and related functionality to the `LadderGame` class. Since this doesn't really make sense for non-matchmaker games.

Reply:
you are probably right. The game class is quite big right now